specification/components/schemas/responses/generatePatientLetter/200Response.yaml

```diff
@@ -0,0 +1,15 @@
+description: Response contains the binary stream of the generated patient letter.
+headers:
+    X-Correlation-Id:
+      $ref: '../../headers/response/CorrelationID.yaml'
+    Content-Disposition:
+      $ref: '../../headers/response/ContentDispositionAttachment.yaml'
+    Content-Length:
+      $ref: '../../headers/response/ContentLength.yaml'
+content:
+  application/pdf:
+    schema:
+      type: string
+      format: binary
+
+          
```

Comment:
the table that this renders in the VDI doesn't make any sense, 
There is just the Name and description headers with one entry " string binary" under the name.

Is the schema: entry required at all ?

Reply:
Yes, unfortunately it does clutter the preview. But the schema is required as this is how you indicate binary file upload/download in openapi3
https://swagger.io/docs/specification/describing-request-body/file-upload/